core/domain/summary_services_test.py

```diff
@@ -196,6 +196,97 @@ def test_get_displayable_exp_summary_dicts_matching_ids(self):
         self.assertDictContainsSubset(expected_summary,
                                       displayable_summaries[0])
 
+    def test_get_displayable_exp_summary_dicts(self):
+        exp_summary_input = exp_services.get_exploration_summaries_matching_ids(
+            [self.EXP_ID_2])
+        exp_summary_input.insert(0, None)
```

Comment:
Here `exp_services.get_exploration_summaries_matching_ids(
            [self.EXP_ID_2])`
returns a list and we need to insert a None object to that list. The method you suggested will create a list in a list.